### DISCUSSION TITLE
When will 2.0.2 Be Released

### DISCUSSION THREAD

**Discussion by Stewart86** [pallets/flask#4244] [Q&A] [answered]
I encountered a bug in the current version with regards to Flask async using `MethodView`. After some digging, I found that this issue is solved and merged into the next release, which is 2.0.2. 

Just wondering when will there be a next release, as the current release was at least a few month back already. Also noting that the 2.0.2 milestone was last updated 20 days ago. Does that means that 2.0.2 is very close to release?

**Comment by davidism** [comment, marked as answer]
When we have time to go through the release process, it will be released. Sorry, we can't give a more precise answer than that.
